src/java/org/apache/cassandra/net/InboundSink.java

```diff
@@ -100,6 +101,10 @@ public void accept(Message<?> message)
         {
             fail(message.header, t);
 
+            // The site throwing AbortedOperationException is responsible for logging it.
+            if (t instanceof AbortedOperationException)
```

Comment:
Does this mean that we'll silently eat any AOEs that aren't caught? Would it be better to let it bubble up to the top without a catch so we can add new catches where required?

Reply:
No, not really. See the exception is passed to the `fail` call above. We're returning it to the client (and there is a test for that!)